src/components/Navigation.vue

```diff
@@ -110,6 +116,14 @@ export default {
 				},
 			})
 		},
+		isFirst(account) {
+			const accounts = this.$store.getters.getAccounts()
+			return account === accounts[1]
+		},
+		isLast(account) {
+			const accounts = this.$store.getters.getAccounts()
```

Comment:
getters are usually not functions :)
https://vuex.vuejs.org/guide/getters.html#property-style-access

Reply:
yep. for consistency reasons I'll shoot a follow-up PR in a sec

Reply:
it's used in the same style in other occasions in this file

Reply:
:ok_woman: